.github/workflows/tests.yaml

```diff
@@ -38,6 +47,9 @@ jobs:
           echo "::group::Wait for snap to complete"
           snap watch --last=install
           echo "::endgroup::"
+      - name: Enable ssh access
+        uses: mxschmitt/action-tmate@v3
+        if: ${{ inputs.enable_ssh_access }}
```

Comment:
Should we set this with `limit-access-to-actor: true`? It concerns me a bit that the default is to allow anyone to connect to the runner.

Reply:
Great idea, I added that flag and updated the documentation.  

I'll test that it is enforced after this gets merged.